### PR TITLE
Tasing and shooting borgs temporarily slows them down

### PR DESCRIPTION
**Pull request by SonixApache**
Intended to bring combat with borgs more in line with combat between carbons: Open with a taser, then fire a bigger gun, or rely on the pain slowdown to get the kill.
Note that this also applies to MoMMIs.
Discussion is very welcome.

🆑 
 - rscadd: Shooting a silicon with a taser now slows them down for ~18 seconds. 
 - rscadd: Shooting a silicon with a high-powered weapon slows them down for ~3 seconds.

### CODE REVIEW

**Comment by aceedex** [comment]
mob/carbon/human/silicon when?

**Comment by Exxion** [comment]
No, shooting them with *anything that deals no damage* slows them for 18 seconds. This includes tasers, but also includes water guns, laser tag guns, the practice laser, temperature guns, etc.

**Comment by SonixApache** [comment]
🤔

**Comment by Exxion** [comment]
Dude that's what the code says, quite clearly

**Comment by SonixApache** [comment]
oversight, i'm on it

**Comment by SonixApache** [comment]
slight adjustment, but oversight fixed

**Comment by Dacendeth** [comment]
laser tag guns slowing borgs 
BRING IT BACK!

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -458,6 +458,17 @@
 /mob/living/silicon/robot/bullet_act(var/obj/item/projectile/Proj)
 	..(Proj)
 	updatehealth()
+	if (movement_speed_modifier > 0.8)
```
Comment (jknpj) [inline code/modules/mob/living/silicon/robot/robot.dm:461]: NO MAGIC NUMBERS

**Review by jknpj — CHANGES_REQUESTED** [review]

**Review by MadmanMartian — CHANGES_REQUESTED** [review]
Remove magic numbers. Wizwazles begone.

**Comment by MadmanMartian** [comment]
Theoryexploit - get tased, then install a VTEC for faster zoom

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 1 above)
Comment (SonixApache) [reply code/modules/mob/living/silicon/robot/robot.dm:461]: i'll get to it, i swear

**Comment by Ephx1** [comment]
rest in piss forever shiticons

**Comment by KingofHONK** [comment]
It's a good idea. Movespeed is terrifyingly strong in combat and often underestimated. 

t. Shitter that doesn't play anymore

**Comment by SonixApache** [comment]
I'LL FIX IT ~~EVENTUALLY~~ NEVER

**Pull request closed without merging**